src/main/java/com/epam/aidial/core/limiter/CompositeRateLimiter.java

```diff
@@ -0,0 +1,44 @@
+package com.epam.aidial.core.limiter;
+
+import com.epam.aidial.core.ProxyContext;
+import lombok.RequiredArgsConstructor;
+
+@RequiredArgsConstructor
+public class CompositeRateLimiter implements RateLimiter {
```

Comment:
Any reason to have 3 new classes right now? You haven't implemented UserRateLimiter yet.

Reply:
fixed